backend/lcfs/web/api/admin_adjustment/schema.py

```diff
@@ -0,0 +1,53 @@
+from lcfs.web.api.base import BaseSchema
+from typing import Optional, List
+from datetime import date, datetime
+
+class AdminAdjustmentStatusSchema(BaseSchema):
+    admin_adjustment_status_id: int
+    status: str
+
+    class Config:
```

Comment:
this code might not be required, as we're already adding it in BaseSchema and also with latest pydantic version, they've have changed the way to add this in dict format.